### ISSUE TITLE
Typos in stm32h503xx.h

### ISSUE THREAD

**Issue by ArkadiuszRaj** [STMicroelectronics/cmsis_device_h5#6]
There are 3 typos in the file stopping compilation. (release 1.3.0)
Namely:
- ADC_AWD3CR_AWD2CH_19 shall be ADC_AWD3CR_AWD3CH_19           
- GPIO_HSLVR_HSLV11_Msk & GPIO_SECCFGR_SEC11_Msk have missing '0' in defintion

I am providing diff to easly locate those changes:
```
diff --git a/Include/stm32h503xx.h b/Include/stm32h503xx.h
index bc59cb0..cbb3320 100644
--- a/Include/stm32h503xx.h
+++ b/Include/stm32h503xx.h
@@ -2480,7 +2480,7 @@ typedef struct
 #define ADC_AWD3CR_AWD3CH_16           (0x10000UL << ADC_AWD3CR_AWD3CH_Pos)    /*!< 0x00010000 */
 #define ADC_AWD3CR_AWD3CH_17           (0x20000UL << ADC_AWD3CR_AWD3CH_Pos)    /*!< 0x00020000 */
 #define ADC_AWD3CR_AWD3CH_18           (0x40000UL << ADC_AWD3CR_AWD3CH_Pos)    /*!< 0x00040000 */
-#define ADC_AWD3CR_AWD2CH_19           (0x80000UL << ADC_AWD3CR_AWD2CH_Pos)    /*!< 0x00080000 */
+#define ADC_AWD3CR_AWD3CH_19           (0x80000UL << ADC_AWD3CR_AWD3CH_Pos)    /*!< 0x00080000 */

 /********************  Bit definition for ADC_DIFSEL register  ****************/
 #define ADC_DIFSEL_DIFSEL_Pos          (0U)
@@ -6456,7 +6456,7 @@ typedef struct
 #define GPIO_HSLVR_HSLV10_Msk               (0x1UL << GPIO_HSLVR_HSLV10_Pos)        /*!< 0x00000400 */
 #define GPIO_HSLVR_HSLV10                   GPIO_HSLVR_HSLV10_Msk
 #define GPIO_HSLVR_HSLV11_Pos               (11U)
-#define GPIO_HSLVR_HSLV11_Msk               (x1UL << GPIO_HSLVR_HSLV11_Pos)         /*!< 0x00000800 */
+#define GPIO_HSLVR_HSLV11_Msk               (0x1UL << GPIO_HSLVR_HSLV11_Pos)         /*!< 0x00000800 */
 #define GPIO_HSLVR_HSLV11                   GPIO_HSLVR_HSLV11_Msk
 #define GPIO_HSLVR_HSLV12_Pos               (12U)
 #define GPIO_HSLVR_HSLV12_Msk               (0x1UL << GPIO_HSLVR_HSLV12_Pos)        /*!< 0x00001000 */
@@ -6506,7 +6506,7 @@ typedef struct
 #define GPIO_SECCFGR_SEC10_Msk              (0x1UL << GPIO_SECCFGR_SEC10_Pos)       /*!< 0x00000400 */
 #define GPIO_SECCFGR_SEC10                  GPIO_SECCFGR_SEC10_Msk
 #define GPIO_SECCFGR_SEC11_Pos              (11U)
-#define GPIO_SECCFGR_SEC11_Msk              (x1UL << GPIO_SECCFGR_SEC11_Pos)        /*!< 0x00000800 */
+#define GPIO_SECCFGR_SEC11_Msk              (0x1UL << GPIO_SECCFGR_SEC11_Pos)        /*!< 0x00000800 */
 #define GPIO_SECCFGR_SEC11                  GPIO_SECCFGR_SEC11_Msk
 #define GPIO_SECCFGR_SEC12_Pos              (12U)
 #define GPIO_SECCFGR_SEC12_Msk              (0x1UL << GPIO_SECCFGR_SEC12_Pos)       /*!< 0x00001000 */
```

**Comment by ALABSTM** [comment]
Hi @ArkadiuszRaj,

You're right. Thank you for the heads-up. Plus, this looks to affect other CMSIS header files too.

With regards,

**Comment by RJMSTM** [comment]
ST Internal Reference: 196711

**Comment by KRASTM** [comment]
Fixed in commit 08f05f3844f7705c9cc9063efaf2c2719cb40e29
